modules/@ergonode/attributes/src/components/Forms/AttributeForm.vue

```diff
@@ -46,7 +46,22 @@
                     :disabled="!isAllowedToUpdate"
                     :error-messages="errors[groupsFieldKey]"
                     href="attributes/groups/autocomplete"
-                    @input="setGroupsValue" />
+                    @input="setGroupsValue">
+                    <template #placeholder="{ isVisible }">
+                        <DropdownPlaceholder
+                            v-if="isVisible"
+                            :title="placeholder.title"
+                            :subtitle="placeholder.subtitle">
+                            <template #action>
+                                <Button
+                                    title="GO TO ATTRIBUTE GROUPS"
+                                    :size="smallSize"
+                                    :disabled="!isAllowedToUpdate"
+                                    @click.native="onNavigateToAttributeGroups" />
```

Comment:
it would be more readable
`@click.native="onNavigateToAttributeGroups()" `

Reply:
We do not allow to use parentheses with no arguments inside v-on binding